### ISSUE TITLE
Request: split Moby and Non-Moby

### ISSUE THREAD

**Issue by AkihiroSuda** [docker/opensource#113]
@thaJeztah ^^ (not urgent)

**Comment by thaJeztah** [comment]
Yes, I discussed that a while back, but probably good to do so once the TSC becomes effective (so that we can have separate templates for the "docker" open source projects, and "moby". Once that's there, I think we should have a similar repository in the Moby org
